.gitignore

```diff
@@ -1,9 +1,13 @@
-#  Local .terraform directories
-**/.terraform/*
-
-# .tfstate files
+# Compiled files
 *.tfstate
-*.tfstate.*
```

Comment:
What's wrong with `*.tfstate.*`?

Reply:
nothing wrong with it.
the file just gets copied from one repo or another, there they have different implementations (we need to standardize that)